packages/sdk/src/types/VersionedModuleSource.ts

```diff
@@ -41,6 +42,19 @@ export function versionedModuleSourceFromBuffer(
     };
 }
 
+/**
+ * Serialize a versioned module source. Useful when saving to file.
+ * @param {VersionedModuleSource} moduleSource The versioned module source to serialize.
+ * @returns {Uint8Array} Buffer with serialized module source.
+ */
+export function versionedModuleSourceToBuffer(
+    moduleSource: VersionedModuleSource
+): Uint8Array {
+    const versionBytes = encodeWord32(moduleSource.version);
```

Comment:
Shouldn't this be big-endian? That's how we store it elsewhere (e.g., the .v1 format).

As far as I see this function is little endian
```
export function encodeWord32(value: number, useLittleEndian = false): Buffer {
    if (value > 4294967295 || value < 0 || !Number.isInteger(value)) {
        throw new Error(
            'The input has to be a 32 bit unsigned integer but it was: ' + value
        );
    }
    const arr = new ArrayBuffer(4);
    const view = new DataView(arr);
    view.setUint32(0, value, useLittleEndian);
    return Buffer.from(new Uint8Array(arr));
}
```

Unless I'm looking at the wrong function.

Reply:
At least it does not seem to match `versionedModuleSourceFromBuffer`

Reply:
`useLittleEndian` is an optional parameter and defaults to being `false`, so it should be using big-endian.
Also used this to store modules from chain for the tests, so it should work since they are able to parse it.